conf/messages.cy

```diff
@@ -679,34 +679,37 @@ totalExpensesCYA.heading = totalExpensesCYA
 
 ########## Expenses - Repairs and Maintenance ##########
 
-########## Expenses - Repairs and Maintenance ##########
-
 ## Repairs and maintenance amount
-repairsAndMaintenanceAmount.title = How much did you spend on repairs and maintenance?
+repairsAndMaintenanceAmount.heading.individual = How much did you spend on repairs and maintenance?
+repairsAndMaintenanceAmount.heading.agent = How much did your client spend on repairs and maintenance?
 repairsAndMaintenanceAmount.details = Understanding repairs and maintenance expenses
-repairsAndMaintenanceAmount.checkYourAnswersLabel = RepairsAndMaintenanceAmountCheckYourAnswersLabel
-repairsAndMaintenanceAmount.error.nonNumeric = Enter your repairsAndMaintenanceAmount using numbers
-repairsAndMaintenanceAmount.error.required = Enter your repairsAndMaintenanceAmount
-repairsAndMaintenanceAmount.error.wholeNumber = Enter your repairsAndMaintenanceAmount using whole numbers
-repairsAndMaintenanceAmount.error.outOfRange = RepairsAndMaintenanceAmount must be between {0} and {1}
+repairsAndMaintenanceAmount.checkYourAnswersLabel.individual = How much did you spend on repairs and maintenance?
+repairsAndMaintenanceAmount.checkYourAnswersLabel.agent = How much did your client spend on repairs and maintenance?
```

Comment:
We don't need these if they are the same as the headings. This should have failed the MessagesSpec test.

Reply:
MessageSpec was green